src/Microsoft.AspNetCore.Mvc.Abstractions/ApiExplorer/IApiDescriptionProvider.cs

```diff
@@ -26,8 +26,16 @@ public interface IApiDescriptionProvider
         /// </remarks>
         int Order { get; }
 
+        /// <summary>
+        /// Called to create or modify an API description.
```

Comment:
`<see cref="ApiDescription"/>s` (add an "s")

Why not just `Creates or modifies...`? Realize this goes against my `IActionFilter` advice but we're not really wrapping a central call. It's just a chain of providers.